src/preamble.js

```diff
@@ -1004,7 +1004,8 @@ function createWasm(env) {
 
   // prepare imports
   var info = {
-    'env': env
+    'env': env,
+    'wasi_unstable': env
```

Comment:
Can we populate was_unstable with just wasi syscalls?   At least can we have a plan to do that?

Reply:
For cleanliness? I'm not sure if that buys us much.
I don't think we need a specific plan either, because we should either be able to refactor the wasi calls out of env ahead of time, or dynamically create it. Unless I'm missing some reason that makes that hard.

Reply:
Yeah, while there is some theoretical cleanliness with having just the necessary imports on `env` and on `wasi_unstable`, there's no actual benefit, and it's actually larger (extra `=`, `{`, `}` at minimum).